client/src/components/BottomPanel.tsx

```diff
@@ -76,6 +81,29 @@ export const BottomPanel = ({ isOpen, type, onAction }: IPanelProps) => {
 						}}
 					/>
 				);
+			case 'background':
+				return (
+					<>
+						{images.map(({ key, src }) => (
+							<div key={key} className="bottom-panel-background">
+								<IconButton onClick={() => {
+									clearTimeout(timers[0]); // clear any existing timers
+									onAction('background', key); //send background image to app.tsx
+									currentTimer = window.setTimeout(() => { // get current timer id
+										console.log('in setTimeout');
+										onAction('background', 'undefined'); // send undefined image to app.tsx after 1 minute
+									}, 60000);
+									timers[0] = currentTimer; // store current timer id for clearing if new image is selected
+								}}>
+									<Avatar 
+										variant="rounded"
+										src={ src }
+									/>
+								</IconButton>
+							</div>
+						))}
```

Comment:
logic should be handled in another component like with Gifs and Chat
each component should ideally be responsible for only one thing